src/client/components/DocumentList/DocumentLink.module.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../styles/_global.scss';
+
+// DocumentLink
+.DocumentLinkWrap {
+  position: relative;
+  display: block;
+  padding-left: 1.8rem;
```

Comment:
dat ik weet gebruikt het design system van amsterdam eenheden die te delen zijn door 4. (1.8 * 16px) is dat niet right?

Reply:
Dit is nog niet icm Design System